src/alfasim_sdk/_internal/alfacase/case_description.py

```diff
@@ -728,10 +737,10 @@ class LeakEquipmentDescription:
     # fmt: on
     # [[[end]]] (checksum: 28bd1bf52b80d19b34c0fabcd9e93b33)
 
-    discharge_coefficient = attrib_scalar(default=Scalar("dimensionless", 0.85, "-"))
     target_pipe: str = attr.ib(default="", validator=instance_of(str))
     target_position = attrib_scalar(default=Scalar(0.0, "m"))
     target_location = attrib_enum(default=constants.LeakLocation.Main)
+
```

Comment:
Is this blank line intended?

Reply:
Just separating some blocks of variable that have same semantics